src/remapBranch.js

```diff
@@ -0,0 +1,28 @@
+function remapBranch(genItem, getMapping) {
+  const locations = [];
+  let source;
+
+  for (let i = 0; i < genItem.locations.length; i += 1) {
```

Comment:
prefer not using `for of` when not dealing with iterators… typical `for` likely is more performant, and I think in this particular place it can be noticeable